src/Adapter/Container/DoctrineBuilderExtension.php

```diff
@@ -40,25 +41,30 @@
  */
 class DoctrineBuilderExtension implements ContainerBuilderExtensionInterface
 {
-    /** @var string */
-    private $cacheDir;
+    /** @var EnvironmentInterface */
+    private $environment;
 
     /**
-     * @param string $cacheDir
+     * @param EnvironmentInterface $environment
      */
-    public function __construct($cacheDir)
+    public function __construct(EnvironmentInterface $environment)
     {
-        $this->cacheDir = $cacheDir;
+        $this->environment = $environment;
     }
 
     /**
      * {@inheritdoc}
      */
     public function build(ContainerBuilder $container)
     {
-        $yamlParser = new YamlParser($this->cacheDir);
+        $configDirectories = [_PS_ROOT_DIR_ . '/app/config'];
+        $fileLocator = new FileLocator($configDirectories);
+
+        $configLoader = new ConfigYamlLoader($fileLocator);
+        $configPath = sprintf('config_front_%s.yml', $this->environment->getName());
```

Comment:
@PierreRambaud @eternoendless I'm not sure about the `_front` prefix because it is used for legacy containers, which is used on legacy context for both front and admin

So I see two solutions:
- use `_legacy` prefix
- add three new files with `_admin` prefix which will be the same as the front ones, but it complicates the config files even more

Note: this legacy admin container is supposed to be removed in 178 because only the Symfony container will be available in BO, so maybe we can accept this for one version Starting 178 the `_front` files and legacy container will only be used for front

Reply:
:+1: for the `_legacy` prefix and the note :)